.golangci.yml

```diff
@@ -164,9 +164,11 @@ issues:
     - path: internal/app/machined/pkg/system/services
       linters:
         - dupl
-    - path: pkg/config/types/v1alpha1
+
+    # pkg/machinery module
+    - path: config/types/v1alpha1
```

Comment:
`pkg/machinery/config/types/v1alpha1` does not work – golangci-lint uses paths within the module